web/static/css/app.css

```diff
@@ -278,3 +278,9 @@ code {
 .b--lm-green {
   border-color: #81c03d;
 }
+
+@media only screen and (min-width: 950px) {
+  #alphasection button {
+    bottom: -30px;
```

Comment:
these kinds of "_negative_" numbers should always be accompanied by an in-line comment  clarifying _why_ we are applying the style.
While I'm _OK_ with merging this for the Demo, I think we need a _process_ for this.
cc: @iteles